src/asyncrpcoperation.cpp

```diff
@@ -27,35 +27,26 @@ std::map<OperationStatus, std::string> OperationStatusMap = {
 /**
  * Every operation instance should have a globally unique id
  */
-AsyncRPCOperation::AsyncRPCOperation() : error_code_(0), error_message_() {
+AsyncRPCOperation::AsyncRPCOperation(std::shared_ptr<CWallet> const wallet) :
+        m_pwallet(wallet), error_code_(0), error_message_()
+{
     // Set a unique reference for each operation
     boost::uuids::uuid uuid = uuidgen();
     id_ = "opid-" + boost::uuids::to_string(uuid);
     creation_time_ = (int64_t)time(NULL);
+
     set_state(OperationStatus::READY);
 }
 
 AsyncRPCOperation::AsyncRPCOperation(const AsyncRPCOperation& o) :
```

Comment:
should consider getting rid of this copy constructor too for same reason

Reply:
done